src/main/java/org/jenkins/plugins/lockableresources/LockableResourcesManager.java

```diff
@@ -164,12 +166,8 @@ public int getFreeResourceAmount(String label) {
     return free;
   }
 
-  public List<LockableResource> getResourcesWithLabel(String label, Map<String, Object> params) {
```

Comment:
Wondering if it is fair to drop the earlier `public` method, there may be pipelines using it (directly calling LRM)... To err on safe side, I'd keep it, just maybe mark as `@Deprecated` or similar.

Reply:
I don't mind keeping it around, the `params` are actually unused - I'd be tempted to remove it but if we're going to remove the whole thing in a few releases, I guess we might as well keep it?